package.json

```diff
@@ -87,32 +89,33 @@
     ]
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "^2.6.2",
-    "@graphql-codegen/typescript": "^2.5.1",
-    "@graphql-codegen/typescript-graphql-request": "^4.4.10",
-    "@graphql-codegen/typescript-operations": "^2.4.2",
+    "@faker-js/faker": "^7.3.0",
+    "@graphql-codegen/cli": "^2.9.1",
+    "@graphql-codegen/typescript": "^2.7.2",
+    "@graphql-codegen/typescript-graphql-request": "^4.5.2",
+    "@graphql-codegen/typescript-operations": "^2.5.2",
     "@storybook/addon-actions": "^6.5.9",
     "@storybook/addon-essentials": "^6.5.9",
     "@storybook/addon-links": "^6.5.9",
-    "@storybook/builder-webpack5": "^6.5.0-rc.1",
-    "@storybook/manager-webpack5": "^6.5.0-rc.1",
+    "@storybook/builder-webpack5": "^6.5.9",
+    "@storybook/manager-webpack5": "^6.5.9",
     "@storybook/node-logger": "^6.5.9",
     "@storybook/preset-create-react-app": "^4.1.2",
     "@storybook/react": "^6.5.9",
     "@tailwindcss/forms": "^0.5.2",
     "@testing-library/cypress": "^8.0.3",
-    "@types/faker": "5.5.3",
     "@xstate/inspect": "^0.7.0",
     "babel-plugin-polyfill-corejs2": "^0.3.1",
-    "cypress": "^10.2.0",
+    "cypress": "^10.3.1",
     "cypress-intellij-reporter": "^0.0.7",
-    "faker": "^5.5.3",
     "fishery": "^2.2.2",
-    "msw": "^0.42.2",
+    "msw": "^0.44.2",
     "npm-run-all": "^4.1.5",
-    "postcss-cli": "^9.1.0",
+    "postcss-cli": "^10.0.0",
     "prettier": "2.7.1",
-    "start-server-and-test": "^1.11.5"
+    "start-server-and-test": "^1.11.5",
+    "husky": "^8.0.0",
+    "precise-commits": "^1.0.2"
```

Comment:
what does precise-commits give us as a pre-commit over just running something like 'yarn prettier --check'

Reply:
Sorry, I thought I committed changes to Readme already, but instead it was shelved.Now, I have pushed the changes to Readme file. Feel free to edit if you need any changes.